de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/reader/ReaderNodeModel.java

```diff
@@ -98,26 +99,29 @@
   private static final PortType[] IN_TYPES = {};
   private static final PortType[] OUT_TYPES = {FskPortObject.TYPE};
 
-  private final ReaderNodeSettings nodeSettings = new ReaderNodeSettings();
+  static final String CFG_FILE = "filename";
+  
+  private final SettingsModelString nodeSettings = new SettingsModelString(CFG_FILE, null);
```

Comment:
We need to rename nodeSettings to filePath as this no longer describes a set of settings but a single setting (path).